ember/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM danlynn/ember-cli:4.4.0-node_18.2 as build
+FROM danlynn/ember-cli:5.4.1 as build
```

Comment:
you are on ember 4.12 but you install 5.4.1 here? Also ember 4 is not tested against node 20 afaik.

Reply:
Reason why is because of this build error
```
eslint-plugin-n@17.3.1: The engine "node" is incompatible with this module. Expected version "^18.18.0 || ^20.9.0 || >=21.1.0". Got "18.16.0"
```
There exists no ember-cli container with node 18.18
The earliest container with a compatible node version is 5.4.1

Reply:
Have you checked the built version of the app as well?

Reply:
Downgraded plugin n to use the correct node version.

Build errors were because of `ember-cli-deploy-smart-compress`. I thought it was a drop in replacement to the non maintaned `ember-cli-deploy-compress`. Probably configured something wrong somewhere.

Have to use a github url, as the latest version which removed `iltorb` is not released. `iltorb` causes errors for higher node versions.

checked the running container and it works correctly now.